src/view/shell/createNativeStackNavigatorWithAuth.tsx

```diff
@@ -138,6 +138,12 @@ function NativeStackNavigator({
 
   return (
     <NavigationContent>
+      {isWeb && !showBottomBar && (
+        <>
+          <DesktopLeftNav />
+          <DesktopRightNav routeName={activeRoute.name} />
```

Comment:
Having the right sidebar before the main content would break the Focus Order guidelines though.

The left nav should definitely be first.

Reply:
Here's my justification for having it go LeftNav > RightNav > Feed.

[WCAG 2.2](https://www.w3.org/TR/WCAG22/)
**[2.1.2: No Keyboard Trap](https://www.w3.org/TR/WCAG22/#no-keyboard-trap)**
If the feed isn't last in the taborder, then the feed becomes a keyboard trap. With 'infinite-scroll' it's impossible to reliably reach the end of it.

**2.4.3: Focus Order**
[From the supplement](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html): 
**Goal**
Keyboard users navigate content in a correct order.
**What to do**
Elements receive focus in an order that preserves meaning.
**Why it's important**
Navigating a site or application with only the keyboard will make sense.

This last point is especially important. "Navigating a site or application with only the keyboard will make sense."

Right now: it doesn't make sense, since it can't be done.

If we do LeftNav > Feed > RightNav, it would make sense (logically) because of our propensity to read from left-to-right, however it's not _possible_ with the feeds configuration set to load infinitely.

I argue that with proper focus indicators, keyboard navigation in LeftNav > RightNav > Feed order would make the MOST sense given the purpose and nature of the feed block. 

Since we cannot disable this feature (the infinite-scroll) with compromising the UX (of everyone, not just sighted users) then LeftNav > RightNav > Feed is arguably the best possible solution.

Reply:
Leave a comment to revert it when skiplinks are added? 

I personally agree with everything you've said.

Reply:
I'll do one better. I'll go add skiplinks into this PR.

Reply:
But of course now I'm thinking that "main content" is going to be the feed, which still renders that RightNav inaccessible. Thoughts? Ideas?

Honestly, why's it even there? 

We could just eliminate it like this...

![Screenshot of BSky's homepage, showing that most of the content in RightNave is redundant, and the rest can easily be placed in other areas for better functionality.](https://github.com/user-attachments/assets/a192ee4d-e3d1-4dfe-8cfc-cb9723f37c5b)